app/models/response_weight.rb

```diff
@@ -0,0 +1,15 @@
+class ResponseWeight < ActiveRecord::Base
+  attr_accessible :response1_id, :response2_id, :weight
+  def self.find_weight(response1, response2)
+    if response1 == response2
+        return 0
+    end
+    query1 = ResponseWeight.where(:response1_id => response1.to_i, :response2_id => response2.to_i)
```

Comment:
Anyway, can't we use this query instead:
where("(response1_id = :response1_id AND respojse2_id = :response2_id) OR (response1_id = :response2_id AND response2_id = :response1_id)", :response1_id => response1.to_i, :response2_id => response2.to_i)